pom.xml

```diff
@@ -32,7 +31,19 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
-	</dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Is this really needed? We get JUnit 5 from the `spring-boot-starter-test` dependency

Reply:
Yeah, I may have imported it unnecessarily. Will remove